taar_lite/recommenders/guid_based_recommender.py

```diff
@@ -145,7 +149,14 @@ def recommend(self, client_data, limit=4):
         result_dict = self._addons_coinstallations.get(addon_guid, {})
 
         # Apply normalization
-        result_dict = norm_method(addon_guid, result_dict)
+        tmp_result_dict = norm_method(addon_guid, result_dict)
+
+        # Augment the result_dict with the installation counts
+        # and then we can sort better
+        result_dict = {}
+        for k, v in tmp_result_dict.items():
+            lex_value = "{0:020.10f}.{1:010d}".format(v, self._guid_rankings.get(k, 0))
```

Comment:
Please add a comment explaining the expected format of float being formed in this line.

Reply:
Comment added